python/tank_vendor/shotgun_authentication/defaults_manager.py

```diff
@@ -18,6 +17,18 @@ class DefaultsManager(object):
     authenticating a user.
     """
 
+    def __init__(self):
+        """
+        Consructor.
+
+        Reads the default host and login from disk.
+        """
+        self._host = session_cache.get_current_host()
```

Comment:
Maybe if session_cache was a class, it could be instantiated here? I am not sure, it's hard to say from just browsing the code, but that's my gut feeling.